bin/smack.py

```diff
@@ -155,6 +221,16 @@ def arguments():
   #     if m:
   #       return args = parser.parse_args(m.group(1).split() + sys.argv[1:])
 
+  if args.svcomp:
```

Comment:
Please replace this code and the `replacer` function by a `svcomp_frontend` function which is invoked when the input file extension is `.i`.